qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -98,6 +98,14 @@
                             cpu_model = host
                         - default_model:
                             cpu_model_flags += ",+pcid,+invpcid"
+                - v_spec_ctrl:
+                    flags = "v_spec_ctrl"
+                    type = x86_cpu_v_spec_ctrl
+                    supported_models = "EPYC-Milan"
+                    boot_option = "spec_store_bypass_disable=on"
```

Comment:
Support no windows guest?

Reply:
> Support no windows guest?

Yes, update and thanks.

Reply:
Do we need restrict linux guest as well? Do old guests work for this case?